File: experiments/020-PythonVsJava/cells/python-pytest_1-Empty_free/2022-1c-recuperatorio-parcial1/20260921-211617-6449/output/project/src/customerimporter/integration_environment.py
from customerimporter.development_environment import DevelopmentEnvironment
from customerimporter.environment import Environment
from customerimporter.persistent_customer_system import PersistentCustomerSystem


class IntegrationEnvironment(Environment):

    # current

    @classmethod
    def is_current(cls):
        return not DevelopmentEnvironment.is_current()

    # customer system

    def create_customer_system(self):
        return PersistentCustomerSystem()

File: experiments/020-PythonVsJava/cells/python-pytest_1-Empty_free/2022-1c-recuperatorio-parcial1/20260921-203929-73794/exercise/java/src/main/java/customerimporter/IntegrationEnvironment.java
package customerimporter;

public class IntegrationEnvironment extends Environment {

    // current

    public boolean isCurrent() {
        return !new DevelopmentEnvironment().isCurrent();
    }

    // customer system

    public CustomerSystem createCustomerSystem() {
        return new PersistentCustomerSystem();
    }
}
